CorsixTH/Lua/graphics.lua

```diff
@@ -430,7 +430,7 @@ function Graphics:loadFont(sprite_table, x_sep, y_sep, ...)
 
   local use_bitmap_font = true
   -- Force bitmap font for the moneybar (Font05V)
-  if not sprite_table:isVisible(46) or load_font == "Font05V" then -- uppercase M
+  if not sprite_table:isVisible(46) or load_font == "Font05V" then -- luacheck: ignore 542
```

Comment:
Why delete the other comment?

Reply:
The next paragraph has the useful information.